providers/google.js

```diff
@@ -1,42 +1,48 @@
-import google from 'googleapis';
+import { google } from 'googleapis';
 import nconf from '../config';
 import User from '../models/user';
 
 const OAuth2 = google.auth.OAuth2;
 
 export default class GoogleProvider {
-
   constructor(token, dce) {
     this.dce = dce;
     this.authLevel = nconf.get('permissions:levels:high');
     this.token = token;
-    this.client = new OAuth2(nconf.get('auth:google:id'), nconf.get('auth:google:secret'));
+    this.client = new OAuth2(
+      nconf.get('auth:google:id'),
+      nconf.get('auth:google:secret')
+    );
   }
 
   verify() {
     return new Promise((resolve, reject) => {
-      // TODO: 'clientId' is 'undefined'; should be 'clientId_'
-      // This may change when upgrading 'googleapis'
-      this.client.verifyIdToken(this.token, this.client.clientId, (err, ticket) => {
-        if (err) {
-          return reject(err);
-        }
-        this.payload = ticket.getPayload();
-        if (this.payload.hd === 'g.rit.edu') {
-          return resolve();
+      this.client.verifyIdToken(
+        {
+          idToken: this.token,
+          audience: this.client.clientId,
```

Comment:
As the TODO above mentions, do we need to update this?

Reply:
The newer version of googleapis has typescript definitions:
`verifyIdToken(options: VerifyIdTokenOptions, callback: (err: Error | null, login?: LoginTicket) => void): void;`
`export interface VerifyIdTokenOptions {
    idToken: string;
    audience: string | string[];
    maxExpiry?: number;
}`
OAth2Client has `    _clientId?: string;` but this isn't exposed for use.
From what I can tell, we've always been sending `undefined` as the audience param, which is kind of strange. 
For general purposes, we can still log in via google oath2 and check the ticket payload to ensure it's a 'g.rit.edu' email, but since we're missing the audience, it isn't restricting the scope of the token.
I can leave the comment in, since the situation hasn't changed, but I think fixing this might be better suited for another PR